tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -450,10 +450,10 @@ public function userSetsOrUpdatesFollowingPasswordForLastLinkShareUsingTheGraphA
 
 	/**
 	 * @param string $sharer
-	 * @param string $shareType (user|group)
-	 * @param string $resource
+	 * @param string $shareType (share|link)
```

Comment:
though `user|group` is not used inside the method. still good to have them
```suggestion
	 * @param string $shareType (user|group|link)
```